crates/oxc_allocator/src/vec2/mod.rs

```diff
@@ -1170,7 +1170,7 @@ impl<'bump, T: 'bump> Vec<'bump, T> {
     /// ```
     #[inline]
     pub unsafe fn set_len(&mut self, new_len: usize) {
-        self.buf.len = new_len;
+        self.buf.len = usize_to_u32(new_len);
```

Comment:
Add to safety comment that `new_len` must be `<= u32::MAX`. `capacity` is always `<= u32::MAX`, and it's in the safety docs that `new_len` must be `<= self.capacity()`, so that's kind of already covered, but I still think it's worth repeating.

Reply:
Yes, I added a safety comment here.